tools/stamp_file.bzl

```diff
@@ -31,20 +29,7 @@ _stamping = rule(
             executable = True,
             cfg = "exec",
             allow_files = True,
-            default = "//stamping:stamping.py",
+            default = "//tools:stamp_file.py",
```

Comment:
Should we wrap these in a `py_binary` ?

I don't think it'll matter for caching, but seems like a good idea to use the same python toolchain as the rest of the build (right now it just finds whatevers in `/usr/bin`